data/json/mutations.json

```diff
@@ -2521,15 +2538,151 @@
         "type" : "mutation",
         "id" : "GIZZARD",
         "name" : "Gizzard",
-        "points" : -5,
-        "description" : "You seem to get full faster now, and food goes through you more rapidly as well.",
+        "points" : 2,
+        "description" : "Your stomach has split into a smaller stomach, and a gizzard. Slightly increases the energy gained through eating, and prevents you from eating as much.",
```

Comment:
Double space here, too. The description is also not quite clear; what exactly is not eating as much? Is it less food per bite, or a smaller stomach capacity, or...?